### PR TITLE
Added drop-down component 

### PR DESCRIPTION
**Pull request by AbigailSalazar**
Hi! I just started learning react but I'm trying to contribute to the project. I made the dropdown component so far. Tell me if you see any problems.

### CODE REVIEW

File: src/components/Dropdown/index.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    title: string;
+    initialCollapsed: boolean;
+    items:React.ReactNode[]
+  }
+  
+  export default function Dropdown({ title, initialCollapsed,items}: DropdownProps) {
+
+    const [collapsed, setCollapsed] = useState(initialCollapsed);
+    const onClick = ()=>{
+        setCollapsed(!collapsed);
```
Comment (HitBox38) [inline src/components/Dropdown/index.tsx:14]: small suggestion:
```tsx
setCollapsed((prev) => !prev)
```

Read more here: https://react.dev/reference/react/useState#setstate-parameters

File: src/components/Dropdown/index.tsx
Comment (HitBox38) [inline src/components/Dropdown/index.tsx:1]: Make sure your prettier formatter works.

**Review by HitBox38 — CHANGES_REQUESTED** [review]
Hi,
Overall this looks good I've a few small suggestions to improve the code.

Thanks for your contribution!

File: src/components/Dropdown/index.tsx
Comment (AbigailSalazar) [reply src/components/Dropdown/index.tsx:1]: I didn't have it, now it works ok 👍

**Review by HitBox38 — APPROVED** [review]

**Comment by HitBox38** [comment]
@AbigailSalazar thanks for your contribution if you're in the discord feel free to tag me for a contributor role.

**Pull request merged**